vllm/worker/cpu_model_runner.py

```diff
@@ -89,6 +94,24 @@ def load_model(self) -> None:
             scheduler_config=self.scheduler_config,
             cache_config=self.cache_config)
 
+        # Apply torch.compile to custom ops
+        from vllm.model_executor.custom_op import CustomOp
```

Comment:
Why do we need lazy import here?

Reply:
For now, we only compile the ```CustomOps```, so import the class for type identification.

Reply:
Oh actually my question was why we import these "lazily".

Reply:
No special reason. I think importing related classes at the local scope will make maintenance more convenient (for example, adding more transformation or moving the procedure to other places).

Reply:
Thanks for the explanation. I personally think it's always good to avoid lazy imports whenever possible, but I agree that it can be a matter of personal preference. I'm ok with keeping it.